app/tests/unit/routes/v1.spec.js

```diff
@@ -15,7 +15,50 @@ describe(`GET ${basePath}`, () => {
     expect(response.statusCode).toBe(200);
     expect(response.body).toBeTruthy();
     expect(Array.isArray(response.body.endpoints)).toBeTruthy();
-    expect(response.body.endpoints).toHaveLength(1);
-    expect(response.body.endpoints).toContain('/hello');
+    expect(response.body.endpoints).toHaveLength(6);
+    expect(response.body.endpoints).toEqual(expect.arrayContaining([
+      '/appConfigForm',
+      '/audit',
+      '/checks',
+      '/docs',
+      '/email',
+      '/kcClientForm'
+    ]));
+  });
+});
+
+
+describe(`GET ${basePath}/docs`, () => {
+  it('should return a redoc html page', async () => {
+    const response = await request(app).get(`${basePath}/docs`);
+
+    expect(response.statusCode).toBe(200);
+    expect(response.text).toContain('<title>GetOK API - Documentation');
+  });
+});
+
+describe(`GET ${basePath}/api-spec.yaml`, () => {
+  it('should return the OpenAPI yaml spec', async () => {
+    const response = await request(app).get(`${basePath}/api-spec.yaml`);
+
+    expect(response.statusCode).toBe(200);
+    expect(response.body).toBeTruthy();
+    expect(response.headers['content-type']).toBeTruthy();
+    expect(response.headers['content-type']).toMatch('application/yaml; charset=utf-8');
+    expect(response.text).toContain('openapi: 3.0.2');
+    expect(response.text).toContain('title: GetOK API');
+  });
+});
+
+describe(`GET ${basePath}/api-spec.json`, () => {
+  it('should return the OpenAPI yaml spec', async () => {
```

Comment:
shouldnt this read:   it('should return the OpenAPI JSON spec', async () => {

Reply:
I'll fix this in the next pr